lemur/common/utils.py

```diff
@@ -493,3 +494,21 @@ def is_json(json_input):
     except ValueError:
         return False
     return True
+
+
+def drop_last_cert_from_chain(full_chain: str) -> str:
+    """
+    drops the last certificate from a certificate chain
+    :param full_chain: string of a certificate chain
+    :return:  string of a new certificate chain, omitting the last certificate
+    """
+    if full_chain == '':
+        return ''
+    full_chain_certs = CERT_PEM_REGEX.findall(full_chain.encode())
+    pem_certificate = OpenSSL.crypto.dump_certificate(
+        OpenSSL.crypto.FILETYPE_PEM,
+        OpenSSL.crypto.load_certificate(
+            OpenSSL.crypto.FILETYPE_PEM, ''.join(cert.decode() for cert in full_chain_certs[:-1])
+        ),
+    ).decode()
+    return pem_certificate
```

Comment:
Is there any pre-existing PEM decoding in the flow(s) where this is used? I'm wondering if it could present any new decoding errors in places that previously wouldn't have noticed them (though that's probably for the best, as valid PEMs seems ideal).

Reply:
thanks for pointing this out, looks like since this treatment is running already in the ACME handler, we should be good
https://github.com/Netflix/lemur/blob/dbb4894de82606a2adadacc66dfd07dffec8eff4/lemur/plugins/lemur_acme/acme_handlers.py#L139-L144